AnnoyancesFilter/sections/widgets.txt

```diff
@@ -497,6 +499,7 @@ miasskiy.ru##.theiaStickySidebar > div.jeg_pb_boxed_shadow
 harpersbazaar.com.ua#?#.content > div:has(> div#SinoptikInformer)
 podillyanews.com#?#.sidebar-box > div.sidebar-title:contains(Погода)
 volyninfa.com.ua#?##sidebar div.widget:has(> div.textwidget > div#SinoptikInformer)
+24ur.com##onl-weather
```

Comment:
24ur.com##onl-poll
24ur.com##onl-joke
24ur.com##onl-weather
these widgets are not annoying and they are first-party, we should not block them